front-end-web/src/Components/Pages/SuccessPage.tsx

```diff
@@ -1,16 +1,19 @@
 import React, { Fragment } from "react";
 import { Props } from "../../@Types";
-
+import { Typography } from "@material-ui/core";
 /**
  * TODO: Add Documentation
  */
+
+var stage = 4;
```

Comment:
Just remove this as it's redundant with the new state that was updated after you committed, otherwise you would be good to go.